src/dlsnode/util/aio/internal/AioScheduler.d

```diff
@@ -92,8 +100,8 @@ class AioScheduler: ISelectEvent
         exception.enforceRetCode!(pthread_mutex_init).call(
                 &this.queue_mutex, null);
 
-        this.ready_queue = new TreeQueue!(SuspendableRequestHandler);
-        this.waking_queue = new TreeQueue!(SuspendableRequestHandler);
+        this.ready_queue = &this.queues[0];
```

Comment:
Looks like something's missing from this commit. Don't you need to new `queues[0]` and `queues[1]`?

Reply:
No, because `TreeQueue` is a struct and it's newed with this class. I've changed this as per @david-eckardt-sociomantic suggestion, I don't have any preferences myself.

Reply:
Ah I see. It being a struct didn't occur to me.